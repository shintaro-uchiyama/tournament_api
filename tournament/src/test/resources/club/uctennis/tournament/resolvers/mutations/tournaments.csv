"id", "title", "subtitle", "date", "image", "description", "create_date", "update_date"
4, "title4", "subtitle4", "2018-08-07 10:00:00", "image4.jpg", "description4", "2018-08-07 10:00:00", null