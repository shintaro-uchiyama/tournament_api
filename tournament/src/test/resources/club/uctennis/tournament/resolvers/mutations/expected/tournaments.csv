"id", "title", "subtitle", "date", "image", "description", "create_date", "update_date"
1, "title1", "subtitle1", "2018-08-07 10:00:00", "image1.jpg", "description1", "2018-08-07 10:00:00", null